base/setup/welcome/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ add_importlibs(welcome gdi32 user32 shell32 msvcrt kernel32 ntdll)
 if(ARCH STREQUAL "i386")
     if(MSVC)
         # NOTE: We cannot use the following command:
-        #   add_target_link_flags(welcome "/SUBSYSTEM:WINDOWS,4.00")
+        #   target_link_options(welcome PRIVATE "/SUBSYSTEM:WINDOWS,4.00")
```

Comment:
I'm all in favor of making this change. But does it in any way relate to fixing clang-cl boot, as its presence in this PR would have you believe?

Reply:
No, it doesn't. I just pushed all the stuff laying around in one PR to check the CI